packages/expo-cli/src/commands/eas-build/build/action.ts

```diff
@@ -91,7 +97,8 @@ async function createBuilderContextAsync(
 async function startBuildsAsync(
   ctx: BuilderContext,
   projectId: string,
-  platform: BuildOptions['platform']
+  platform: BuildOptions['platform'],
+  skipProjectConfiguration: boolean = false
```

Comment:
Opinion: 3 arguments were still acceptable, 4 is already too many. If I was implementing this I'd move `platform` and `skipProjectConfiguration` to `BuilderContext`.